list_book.rb

```diff
@@ -0,0 +1,25 @@
+module ListBook
+  def list_all_books
+    Book.each do |book|
+      puts "Title: #{book.title} | Author: #{book.author}"
+    end
```

Comment:
Option 4: List all books throws an error as well.

![image](https://user-images.githubusercontent.com/38088818/174106406-45c165b7-fa7a-43ce-af84-8539aec67458.png)

- When I created a book in option 2 I was notified that the book was created successfully. Therefore, I should be able to see the book that I created when I list all books.